patch-package
--- a/node_modules/metro/src/JSTransformer/index.js
+++ b/node_modules/metro/src/JSTransformer/index.js
@@ -139,6 +139,8 @@ module.exports = class Transformer {
     /^--heap[_-]growing[_-]percent=[0-9]+$/.test(arg) ||
     /^--max[_-]old[_-]space[_-]size=[0-9]+$/.test(arg));
 
+    execArgv.push("--max-old-space-size=8192");
+
 
     return new Worker(workerPath, {
       computeWorkerKey,
--- a/node_modules/metro/src/defaults.js
+++ b/node_modules/metro/src/defaults.js
@@ -47,7 +47,7 @@ exports.sourceExts = ['js', 'json'];
 
 exports.moduleSystem = require.resolve('./lib/polyfills/require.js');
 
-exports.platforms = ['ios', 'android', 'windows', 'web'];
+exports.platforms = ['ios', 'android', 'windows', 'web', 'desktop'];
 
 exports.providesModuleNodeModules = ['react-native', 'react-native-windows'];
 
