--- /tmp/tmp-status-mobile-6e854428f/tmp.PSruc8WtHx/build.gradle	2024-04-17 11:08:46.857319000 +0200
+++ ./node_modules/react-native-mail/android/build.gradle	2024-04-17 11:09:22.935436792 +0200
@@ -1,7 +1,11 @@
 apply plugin: 'com.android.library'
 
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 android {
-    compileSdkVersion project.hasProperty('compileSdkVersion') ? project.compileSdkVersion : 23
+    compileSdkVersion safeExtGet('compileSdkVersion', 30)
     buildToolsVersion project.hasProperty('buildToolsVersion') ? project.buildToolsVersion : "23.0.1"
 
     defaultConfig {
