# https://github.com/wix/react-native-navigation/issues/7819

--- ./node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/stack/topbar/button/ButtonPresenter.kt
+++ ./node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/stack/topbar/button/ButtonPresenter-patched.kt
@@ -194,8 +194,8 @@

                 class WixAccessibilityDelegateCompat: AccessibilityDelegateCompat(){
                     override fun onInitializeAccessibilityNodeInfo(
-                        host: View?,
-                        info: AccessibilityNodeInfoCompat?
+                        host: View,
+                        info: AccessibilityNodeInfoCompat
                     ) {
                         super.onInitializeAccessibilityNodeInfo(host, info)

@@ -216,4 +216,4 @@
     }


-}
\ No newline at end of file
+}
