diff --git a/node_modules/react-native/local-cli/runIOS/runIOS.js b/node_modules/react-native/local-cli/runIOS/runIOS.js
index dfabb92b8..5b2714729 100644
--- a/node_modules/react-native/local-cli/runIOS/runIOS.js
+++ b/node_modules/react-native/local-cli/runIOS/runIOS.js
@@ -283,6 +283,7 @@ function buildProject(
       configuration,
       '-scheme',
       scheme,
+      '-UseModernBuildSystem=NO',
       '-destination',
       `id=${udid}`,
       '-derivedDataPath',
